src/store/modules/organizations.js

```diff
@@ -15,20 +24,41 @@ export default {
     setOrgs(state, organizations) {
       state.organizations = organizations;
     },
+    setUserOrgs(state, orgs) {
+      state.userOrganizations = orgs;
+    },
   },
   actions: {
     async fetchOrgs({ commit }) {
       const res = await fetchOrgs();
       commit("setOrgs", res);
     },
+    async fetchOrgsById({ commit }, orgId) {
```

Comment:
This is returning the resources availability of an organization. The current name is misleading.

Reply:
fixed